libs/components/list-builder-view-checklist/src/lib/modules/list-view-checklist/fixtures/list-view-checklist-toolbar.component.fixture.ts

```diff
@@ -32,7 +32,7 @@ export class ListViewChecklistToolbarTestComponent {
   }
 
   public changeVisibleItems(checked: boolean): void {
-    const checkbox = new SkyCheckboxChange();
+    const checkbox = { checked } as SkyCheckboxChange;
```

Comment:
You could combine all this into one statement since you no longer have to do the two step construct an object/assign a property to it logic.

```
this.checklist.changeVisibleItems({
  checked
});
```